package.json

```diff
@@ -5,11 +5,11 @@
   "main": "index.js",
   "scripts": {
     "start": "webpack-dev-server -d --progress --inline --hot --host 0.0.0.0",
-    "build:debug": "webpack -d --progress --env.verbose",
-    "build": "webpack --progress",
-    "build:prod": "webpack -p --progress --env.NODE_ENV=production --env.BUILD_PDF",
+    "build:debug": "webpack --mode=development -d --progress --env.verbose",
+    "build": "webpack --mode=development --progress",
+    "build:prod": "webpack --mode=production -p --progress --env.BUILD_PDF",
     "build:analyse:": "webpack --progress --profile --json > stats.json",
```

Comment:
Selv om `--mode=production` er default, så hadde det sikkert vært lurt å legge det på for `build:analyse` også. Og, er det meningen at det skal være en `:` til slutt der? `build:analyse:`?